frontend/src/components/containers/ContainerEditContent.js

```diff
@@ -14,7 +14,7 @@ import {container as EMPTY_CONTAINER} from "../../models";
 import api, {withToken} from "../../utils/api";
 
 const requiredRules = [{ required: true, message: 'Missing field' }]
-const barcodeRules = [{ pattern: /^[a-zA-Z0-9.\-_]{1,199}$/ }]
+const barcodeRules = [{ pattern: /^[^$]{1,199}$/ }]
```

Comment:
So this means a barcode is anything except a dollar sign, right?

Reply:
Yup it had to be changed, the backend was correct but not the frontend. It is already on master, as part of another PR

Reply:
Yeah sorry i rebased my code to import changes to cypress